sunpy/bin/deactivate

```diff
@@ -0,0 +1 @@
+/home/anomaly_/miniconda2/bin/deactivate
```

Comment:
Will remove these files, but how do I remove the astropy helpers? Without making a hard-reset, how do I just remove that part?